awscli/topics/s3-config.rst

```diff
@@ -110,6 +110,7 @@ multipart_threshold
 -------------------
 
 **Default** - ``8MB``
+**Minimum** - ``5MB``
```

Comment:
We should probably update these docs and add a note that this will not apply to downloads.